src/Cafe/OS/libs/nn_fp/nn_fp.cpp

```diff
@@ -472,6 +480,14 @@ namespace nn
 			return ipcCtx->Submit(std::move(ipcCtx));
 		}
 
+		nnResult GetMyComment(iosu::fpd::FPDPreference* myPreference)
+		{
+			FP_API_BASE();
+			auto ipcCtx = std::make_unique<FPIpcContext>(iosu::fpd::FPD_REQUEST_ID::GetMyComment);
+			ipcCtx->AddOutput(myPreference, sizeof(uint16be)*0x12);
```

Comment:
Can you use `MY_COMMENT_LENGTH` here instead of hardcoding 0x12. The definition of `MY_COMMENT_LENGTH` would have to be moved from `CallHandler_GetMyComment` into iosu_fpd.h